arches/app/models/graph.py

```diff
@@ -2636,6 +2627,16 @@ def promote_draft_graph_to_active_graph(self):
             ]
         ]
 
+        serialized_draft_graph["functions_x_graphs"] = serialized_source_graph[
+            "functions_x_graphs"
+        ]
+
+        for function_x_graph in serialized_draft_graph["functions_x_graphs"]:
+            if "config" in function_x_graph:
+                function_x_graph["config"] = self.replace_config_ids(
```

Comment:
shouldn't need replace config ids here? Since it's just going from source_graph to source_graph?

Reply:
yeah that's fair.